docs/guide/shop_configuration.md

```diff
@@ -0,0 +1,101 @@
+
+For some of the setting, you can configure them differently for different SiteAccesses.
+
+![](img/shop_configuration_settings.png)
+
+!!! caution
+
+    Settings defined in the Back Office always override the configuration in YAML files.
+
+## Catalog
+
+|Name|Description|
+|--- |--- |
+|Layout product category page|Layout for the product category page.</br>`category` - show category boxes only</br>`product_list` - show products only</br>`both` - show both categories and products.|
+|Last viewed product limit|The maximum number of last viewed products to store (per user).|
+|Product description character limit|Number of characters from the Subtitle Field that are visible as description in the list views and search results.|
+
+## Price
+
+|Name|Description|
+|--- |--- |
+|Automatic currency conversion|When enabled, calculates the price using the configured conversion rate if no price is set up for the current currency.|
+|Currency conversion rate|The conversion rate between currencies.|
+|Default currency|Used as a fallback when SiteAccess-specific currency is not set.|
+|Base currency|Base currency of the shop, used for the Fields "Product unit price" and "Fallback shipping price". The base currency is used for the automatic currency conversion.|
+|Price providers|Price engines used for generating price and stock information in different parts of the shop. This configuration works as a chain, so if the first engine fails, the second one is used (e.g. if ERP is not available).|
+
+## Advanced catalog features 
+
+|Name|Description|
+|--- |--- |
+|User interface for ordering variants|User interface used for ordering variants on the product detail page. `B2B` is optimized for ordering more than one product at once. `B2C` enables ordering for one product only, but is more user friendly.
+
+## ERP
+
+[ERP integration](erp_integration/erp_integration.md) requires a Web.Connector license or another webservice interface between ERP and the shop.
+
+|Name|Description|
+|--- |--- |
+|Default Country for Template Debitor|Country used as a default.|
+|Login with Customer Number|When enabled, the login process includes a field for providing the customer number.|
+|Use a template debitor number for this shop|When enabled, a template debitor customer number is used if a customer does not have a customer number from the ERP. Template debitor customer numbers can be defined for each country.|
+|Use a template contact number for this shop|When enabled, a template contact number is used if a customer does not have a customer and contact number from the ERP. Template contact numbers can be defined for each country.|
+|Price requests without customer number|When enabled, a price request is sent to the ERP without customer number. A template debitor is used to calculate prices.|
+|Recalculate prices using the ERP after (seconds)|How long information from the ERP is cached to reduce the traffic towards the ERP. Use a "1 hour" syntax.|
```

Comment:
"1 hour" syntax

What does it mean? I wasn't able to find a definition. Is it "<number> hour"?
A similar statement can be found here, with a possible error (hour/hours):
https://doc.ibexa.co/projects/userguide/en/latest/shop_administration/configuration_settings/#basket

Reply:
The syntax is (presumably) "1 hour", "3" hours", "1 day", to be verified with the rest of ERP section.